JavaScriptSPA/authConfig.js

```diff
@@ -1,21 +1,25 @@
+
+// Config object to be passed to Msal on creation.
+// For a full list of msal.js configuration parameters, 
+// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_authenticationparameters_.html
```

Comment:
Couple things: 
- msal-js configuration is here: `https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_configuration_.html`
- msal-js request(AuthneticationParameters) is here: `https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_authenticationparameters_.html`

Please add both in the comments and teh respctive content